lib/iex/lib/iex.ex

```diff
@@ -766,9 +767,8 @@ defmodule IEx do
 
   """
   @doc since: "1.5.0"
-  def break!(module, function, arity, stops \\ 1) when is_integer(arity) do
-    IEx.Pry.break!(module, function, arity, stops)
-  end
+  @spec break!(module, atom, arity, non_neg_integer) :: IEx.Pry.id()
+  defdelegate break!(module, function, arity, stops), to: IEx.Pry
```

Comment:
This is changing the public API because it is dropping the default `stops` argument.

Reply:
I would drop the changes in this file, the other changes are good to go! Good catch!

Reply:
True. I missed the default argument

Reply:
I reduced the guard changes to a minimum. Please have a look. I'm using default arg with defdelegate, and adding specs since they are not getting imported by ExDoc. Please let me know if that's fine or everything should be dropped